libs/core/algorithms/tests/unit/unseq_algorithms/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(tests "")
```

Comment:
This file misses Copyright, License, and SPDX-License-Identifier. It also misses an endline on the last line.